docs/sources/release-notes/v2.2.md

```diff
@@ -25,15 +25,27 @@ This feature still needs additional testing; we do not recommend using it in pro
+  - The Helm chart now supports [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift).
+  - The Helm chart can now easily deploy [Grafana Agent](https://github.com/grafana/agent) in order to scrape metrics and logs from all Mimir pods, and ship them to a remote store, which makes it easier to monitor the health of your Mimir installation. For more information, see [Collecting metrics and logs from Grafana Mimir]({{< relref "../operators-guide/monitoring-grafana-mimir/collecting-metrics-and-logs.md" >}}).
+  - The Helm chart now enables multi-tenancy by default. This makes it easy for you to add tenants as you grow your cluster. You can take advantage of Mimir's per-tenant quality-of-service features, which improves stability and resilience at high scale. To learn more about how multi-tenancy in Mimir works, see [Grafana Mimir authorization and authentication]({{< relref "../operators-guide/securing/authentication-and-authorization" >}}). This change is backwards-compatible. To read about how we implemented this, see [#2117](https://github.com/grafana/mimir/pull/2117).
+  - We have significantly improved the configuration experience for the Helm chart, and here are a few of the most salient changes:
+    - We've added an `extraEnvFrom` capability to all Mimir services to enable you to inject secrets via environment variables.
+    - We've made it possible to globally set environment variables and inject secrets across all pods in the chart using `global.extraEnv` and `global.extraEnvFrom`. Note that the memcached and minio pods are not included.
+    - We've switched the default storage of the Mimir configuration from a `Secret` to a `ConfigMap`, which makes it easier to quickly see the differences between your Mimir configurations between upgrades. We especially like the [Helm diff](https://github.com/databus23/helm-diff) plugin for this purpose.
+    - We've added a `structuredConfig` option, which allows you to overwrite specific key-value pairs in the `mimir.config` template, which saves you from having to maintain the entire `mimir.config` in your own `values.yaml` file.
+    - We've added the ability to create global pod annotations. This unlocks the ability to trigger a restart of all services in response to a single event, such as the update of the secret containing Mimir's storage credentials.
+  - We've set the chart to disable `-ingester.ring.unregister-on-shutdown` and `-distributor.extend-writes`, for a smoother upgrade experience. Rolling restarts of ingesters are now less likely to cause spikes in resource usage.
+  - We've improved the documentation for the Helm chart by adding a [Getting started with Mimir using the Helm chart]({{< relref "../operators-guide/deploying-grafana-mimir/getting-started-helm-charts/_index.md" >}}).
+  - We've added a smoke test for your Mimir cluster to help catch errors immediately after you install or upgrade Mimir via the Helm chart.
```

Comment:
That's a lot of changes to point out in this section in my opinion. We still refer people to the upgrading instructions and full changelog, so should we perhaps only pick most important changes here?

/cc @dimitarvdimitrov

Reply:
fair point, it's taking about a 1/3 of the full release notes. I will trim it down